### DISCUSSION TITLE
Monting mulitple folders into Backup folder

### DISCUSSION THREAD

**Discussion by akociok** [JonathanTreffler/backblaze-personal-wine-container#75] [Q&A] [answered]
Hi, 

I have set the container up with the excellent guide. Works like a charm as long as I just put filed into the backup folder. I thought I could just mount -b my data folders into it, as My data is scattered across 3 harddrive in the same system.
If I do so blackbaze does not seem to see any of the files and folders withing.

Is this behaviour expected? do I have to set the backup folder to the individual first folder the hardrive is mounted in?

Thanks for your help.

**Comment by JonathanTreffler** [comment, marked as answer]
You can repeat the relevant parts of the guide to add multiple backed up drives besides drive_d, just go alphabetically drive_e, drive_f ...

**Reply by wb6vpm** [reply]
So, if I am understanding correctly (and forgive me, I am a complete n00b when it comes to Docker, so if this is a stupid clarification, feel free to tell me so), you could do it via updating the installation script like this:
```
docker run \
    -p 8080:5800 \
    --init \
    --name backblaze_personal_backup \
    -v "[backup folder1]/:/drive_d/" \
-->>    -v "[backup folder2]/:/drive_e/" \
...
    -v "[backup folderX]/:/drive_X/" \ <<--
    -v "[config folder]/:/config/" \
    tessypowder/backblaze-personal-wine:latest
```

To clarify however, how would you add additional drives after the fact?

**Reply by JonathanTreffler** [reply]
> you could do it via updating the installation script like this:

exactly

> To clarify however, how would you add additional drives after the fact?

You need to delete and re-create a docker container to modify it. Dockers have internal storage, but everything, that needs to be permament needs to be saved in a volume (the -v parts of the command) and this docker also does that. So if you create a new docker using the same volume mapping for /config/ it will still be the same Backblaze instance, even though the docker is new.
